server/constants/constants.go

```diff
@@ -0,0 +1,46 @@
+package constants
+
+import "time"
+
+const (
+	APIErrorIDNotConnected = "not_connected"
+	// TokenTTL is the OAuth token expiry duration in seconds
+	TokenTTL = 10 * 60
```

Comment:
How is this in seconds?
```suggestion
	// TokenTTL is the OAuth token expiry duration in minutes
	TokenTTL = 10 * time.Minute
```

Reply:
Is there any reason for this? As in `KVSetWithExpiry` we have to pass this in seconds.

Reply:
How many seconds is this? 600?

Reply:
Yes

Reply:
```suggestion
	// TokenTTL is the OAuth token expiry duration in seconds
	TokenTTL = 600 * time.Second
```

Reply:
I think we should not do this because `time.Second` is returning `1000000000`

Reply:
It's working fine in my system. Let's discuss this.